src/main/window.main.ts

```diff
@@ -0,0 +1,21 @@
+import { ElectronConstants } from 'jslib/electron/electronConstants';
+
+import { MessagingService } from 'jslib/abstractions/messaging.service';
+import { StorageService } from 'jslib/abstractions/storage.service';
+
+import { WindowMain as BaseWindowMain } from 'jslib/electron/window.main';
+
+export class WindowMain extends BaseWindowMain {
```

Comment:
We don't need a new window.main class like this. Your original solution was better. Just send a message from the angular components. We don't want angular components in the renderer process having a dependency on something related to the electron main process.

Reply:
Sure.